src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/test_acs_client.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+import unittest
+import mock
```

Comment:
`mock` module is deprecated in azure cli. see https://github.com/Azure/azure-cli/pull/19024